### DISCUSSION TITLE
How to setup the project ?

### DISCUSSION THREAD

**Discussion by MabudAlam** [CCExtractor/beacon#29] [Q&A]
I have created the .env file but the I don't which API  to use and what the server's endpoint.
Can anyone guide me?

**Comment by Arindam2002** [comment]
I'm having the same issue and because of that the app won't run.

![image](https://user-images.githubusercontent.com/79254056/177053929-b2ad1b70-2cdc-45e1-b0bf-de27e91cadfd.png)

@adarsh-technocrat can you help me out here?

**Reply by ItsAdityaKSingh** [reply]
You haven't put the environment variables right in the .env file. You need to set proper variables in the fields. Recheck the steps on Readme.md and if you face issues, let me know again!
